### ISSUE TITLE
Working with Docker

### ISSUE THREAD

**Issue by timhuijser** [appbaseio/dejaVu#86]
When using the Docker image the client will fail while setting up the interface. The client is successfully retrieving data from the index I filled out, CORS is working as far as I can tell.

What goes wrong is a call made to `https://accapi.appbase.io/user`. In my logs I can see 

```
XMLHttpRequest cannot load https://accapi.appbase.io/user. No 
'Access-Control-Allow-Origin' header is present on the requested resource. 
Origin '<my-dev-domain>' is therefore not allowed access. 
The response had HTTP status code 401.
```

The content of the response is
`{"message":"Not Logged In, No Token"}`

Would be awesome if anyone could give an answer to why this call is made and either how to disable it from doing so or how I can make it work. At first I thought this client was to be used separately from Appbase, is that not the case?

Thanks!

**Comment by siddharthlatest** [comment]
@timhuijser Which port are you using dejavu from? Also, where in the interface does this happen - Are you using the data browser view or the importer view?

**Comment by timhuijser** [comment]
@siddharthlatest Thank you for your fast reply. I am using dejavu from port 1358, I'm accessing it on `<my-dev-domain>:1358/`. I'm using the browser view and the issue is after filling out the index name and the ES target.

See screenshot: http://imgur.com/a/Q9tvX

**Comment by siddharthlatest** [comment]
@timhuijser I tested with `appbaseio/dejavu:0.13.0` (also the latest) and do not see that request being made. Which tag are you using, or are you building from source?

**Comment by timhuijser** [comment]
I was using `appbaseio/dejavu:latest`, when trying `appbaseio/dejavu:0.13.0` I'm getting the same error. While digging around I saw [this code](https://github.com/appbaseio/dejavu/blob/40e0ceac3261dcdebef23f9a4af4bc9817cb69fd/live/src/js/features/Importer.js#L31), which looks like the cause of the call. Any idea why this call is made?

**Comment by siddharthlatest** [comment]
We added an import data functionality recently, where we need to log in a user to show the app to import to. It seems there is a side effect of that in the data browser view, which shouldn't ideally require a login.

Thanks for pointing this out, I would recommend trying 0.12.0 version on docker in the meanwhile.

**Comment by timhuijser** [comment]
Thank you for the explanation, I will try the previous version!

**Comment by siddharthlatest** [comment]
Update: Able to replicate the issue, will be fixed in v0.14.0. Thanks for pointing out @timhuijser.

**Comment by siddharthlatest** [comment]
This has been fixed in the `dev` branch.
